scripts/NameGenerator/NameGenerator.gml

```diff
@@ -0,0 +1,6 @@
+// Script assets have changed for v2.3.0 see
+// https://help.yoyogames.com/hc/en-us/articles/360005277377 for more information
+function NameGenerator() constructor
```

Comment:
```suggestion
function NameGenerator(names_array) constructor
```

alternate is to pass in the string for the path, but this would be useful for making things a bit more generic

Reply:
did something bit similar in `JsonFileListLoader()`, I basically extracted file loading and parsing onto a separate struct, so that the name generators can simply supply a path a list of properties to list

Reply:
N/A anymore, I chose a different approach